backend/src/graphql/resolver/ContributionMessageResolver.ts

```diff
@@ -149,8 +144,7 @@ export class ContributionMessageResolver {
       await queryRunner.commitTransaction()
     } catch (e) {
       await queryRunner.rollbackTransaction()
-      logger.error(`ContributionMessage was not successful: ${e}`)
-      throw new Error(`ContributionMessage was not successful: ${e}`)
+      throw new LogError('ContributionMessage was not sent successfully', e)
```

Comment:
May be at this point it is too few information? Should the original error message be passed to the frontend as well?

Reply:
I agree, fixed